static/scss/partials/main.scss

```
@@ -1032,6 +1032,7 @@ input::placeholder,
 .relay-email-address-label:placeholder-shown {
   color: var(--grey50);
   opacity: 1;
+  width: 192px !important;
```

Is the `!important` tag necessary here? 
```suggestion
  width: 192px !important;
```

---

Also, there is a token, `$layout-2xl` that is `192px` wide. We might use that instead? (That may not be the most semantically correct use of the token)

---

Hmm, the `192px` is more of a Magic Number fix for the length of the placeholder, and from my understanding `$layout-2xl` should be reserved for spacing between elements.

---

Well reasoned. Good to go!